src/ExpandObjects/epfilter.f90

```diff
@@ -23467,6 +23469,10 @@ SUBROUTINE CreateFanCoil
     capacityControlKind = capctrlVarFanVarFlow
   ELSEIF (SameString(FldVal(base +  fczCapCtrlTypeOff),'VariableFanConstantFlow')) THEN
     capacityControlKind = capctrlVarFanConstFlow
+!  ELSEIF (SameString(FldVal(base +  fczCapCtrlTypeOff),'MultiSpeedFan')) THEN
+!    capacityControlKind = capctrlMultiSpeedFan
+  ELSEIF (SameString(FldVal(base +  fczCapCtrlTypeOff),'ASHRAE90VariableFan')) THEN
```

Comment:
I thought I added a new key here in the idd. I don't see it now so HVAC template for fan coil will not be able to create the SZVAV model (yet). These changes shouldn't hurt anything but SZVAV won't work yet. Also, the SZVAV model for UnitarySystem and PackagedTerminalUnit will also need IDD changes since the HVAC template model for these will need a key to know when to create the SZVAV model. I think I'll wait on this since there is so much other work to do.